stable/fairwinds-insights/README.md

```diff
@@ -144,13 +144,15 @@ See [insights.docs.fairwinds.com](https://insights.docs.fairwinds.com/technical-
 | ingress.starPaths | bool | `true` | Certain ingress controllers do pattern matches, others use prefixes. If `/*` doesn't work for your ingress, try setting this to false. |
 | ingress.separate | bool | `false` | Create different Ingress objects for the API and dashboard - this allows them to have different annotations |
 | ingress.extraPaths | object | `{}` | Adds additional path ie. Redirect path for ALB |
+| postgresql.fullnameOverride | string | `"insights-postgresql"` |  |
```

Comment:
I had to hardcode Postgres service name so it's known (not dynamic) ahead of time for temporal